web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  MassaToken,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Selector,
+  Identicon,
+} from '@massalabs/react-ui-kit';
+import { formatStandard } from '../../../utils/MassaFormating';
+import { useResource } from '../../../custom/api';
+import { AccountObject } from '../../../models/AccountModel';
+
+function AccountSelect({ ...props }) {
+  const { modalAccounts, setModalAccounts, setRecipient, account } = props;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const otherAccounts = accounts.filter(
+    (acnt: AccountObject) => acnt.nickname !== account.nickname,
+  );
+
+  function setRecipientAndClose(account: AccountObject) {
+    setRecipient(account.address);
+    setModalAccounts(false);
+  }
+
+  return (
+    <PopupModal
+      fullMode={true}
+      onClose={() => setModalAccounts(!modalAccounts)}
+    >
+      <PopupModalHeader>
+        <label className="mas-title">My accounts</label>
+      </PopupModalHeader>
+      <PopupModalContent>
+        <div>
+          {otherAccounts.map((account: AccountObject) => (
+            <div className="pb-4">
```

Comment:
Why not in selector this className!?

Reply:
we can't add className to the selector because it will overwrite current classes